listenbrainz/db/tests/test_lastfm_user.py

```diff
@@ -63,5 +56,9 @@ def test_user_load_by_id(self):
         self.assertDictEqual(user.__dict__, self.user.__dict__)
 
     def test_user_get_play_count(self):
+        date = datetime(2015, 9, 3, 0, 0, 0)
```

Comment:
The test had started failing for me randomly :(

I had added the reason for moving this block to the commit message https://github.com/metabrainz/listenbrainz-server/pull/212/commits/7056af289d1f21107c1b823d4cec27a17f6acdea